recipe/meta.yaml

```diff
@@ -69,3 +69,4 @@ extra:
   recipe-maintainers:
     - jan-janssen
     - ericpre
+    - stukowski
```

Comment:
@stukowski, a conda-forge practise is to get people confirmation to be recipe maintainer! Can you please confirm?

Reply:
You have my confirmation. Thank you.